clang/test/Driver/sycl-windows-device-filename.cpp

```diff
@@ -0,0 +1,20 @@
+// Test valid file names from -device values for GPU
+// REQUIRES: system-windows
+
+// RUN: %clang -### --target=x86_64-pc-windows-msvc -fsycl \
+// RUN:   -fsycl-targets=spir64_gen --offload-new-driver \
+// RUN:   -Xsycl-target-backend "-device arch1:arch2" %s 2>&1 \
```

Comment:
What is the behavior when the device arch values are comma separated or space separated?

Reply:
The strings are not modified.  Now, using commas is fine for file names AFAIK, but this could throw a wrench into the `arch` settings for the `clang-offload-packager`.  

@asudarsa, any thoughts on how we can handle comma separated arch values when passed in like `-device arch1,arch2,arch3` and is embedded in the `clang-offload-packager` image-file stringmap?  As the values are comma separated I think it would be parsed incorrectly.

Reply:
@srividya-sundaram, I have added some additional changes to convert commas.  Additional side discussions to be had with @asudarsa to resolve other issues with the packager.